src/protobot_planner/nodes/test.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3
+
+from actionlib_msgs.msg import GoalStatusArray
```

Comment:
?

Reply:
think this is @frostfireftw 's testing? will remove in the next commit?